docs/asyncio_api.rst

```diff
@@ -15,11 +15,11 @@ You can use the method ``request_raw`` to perform individual requests:
     client = AsyncZyteAPI(api_key="YOUR_API_KEY")
 
 
-    async def single_request(url):
-        return await client.get({"url": url, "browserHtml": True})
+    async def main():
```

Comment:
I really like this change of the function name, makes it clear that this is just scaffolding to make the example run 👍